From 2b273ba9896b1e9fb1e4edd2f11d1514a905c968 Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Tue, 03 Jun 2025 13:01:30 +0300
Subject: [PATCH] [SW-221571] common: add missing is present impls

Add implementations of all is_present
functions for each device that may not be present
to both gpu cards.

Change-Id: I503fb28c233bf364f1fee3bc5ee6d4e85bbe2094
Signed-off-by: Tal Yacobi <tyacobi@habana.ai>
---

diff --git a/drivers/misc/common_utils/gpu/fs1/common_utils.c b/drivers/misc/common_utils/gpu/fs1/common_utils.c
index 859137e..1cc49fd 100644
--- a/drivers/misc/common_utils/gpu/fs1/common_utils.c
+++ b/drivers/misc/common_utils/gpu/fs1/common_utils.c
@@ -155,3 +155,48 @@
 addr_cntrl:
 	CFG_WREG32((offset + reg), en ? 1 : 0);
 }
+
+bool hl_is_tmp464_present(void)
+{
+	return false;
+}
+
+bool hl_is_tmp432_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpe1a2g5b_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpe_present(void)
+{
+	return false;
+}
+
+bool hl_is_mp2891_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpp1100_present(void)
+{
+	return false;
+}
+
+bool hl_is_adc121c027_present(void)
+{
+	return false;
+}
+
+bool hl_is_ina236_0_present(void)
+{
+	return false;
+}
+
+bool hl_is_sfp_present(void)
+{
+	return false;
+}
diff --git a/drivers/misc/common_utils/gpu/jgs/common_utils.c b/drivers/misc/common_utils/gpu/jgs/common_utils.c
index cfe3ff1..6b69960 100644
--- a/drivers/misc/common_utils/gpu/jgs/common_utils.c
+++ b/drivers/misc/common_utils/gpu/jgs/common_utils.c
@@ -98,3 +98,48 @@
 {
 	return 0;
 }
+
+bool hl_is_tmp464_present(void)
+{
+	return false;
+}
+
+bool hl_is_tmp432_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpe1a2g5b_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpe_present(void)
+{
+	return false;
+}
+
+bool hl_is_mp2891_present(void)
+{
+	return false;
+}
+
+bool hl_is_xdpp1100_present(void)
+{
+	return false;
+}
+
+bool hl_is_adc121c027_present(void)
+{
+	return false;
+}
+
+bool hl_is_ina236_0_present(void)
+{
+	return false;
+}
+
+bool hl_is_sfp_present(void)
+{
+	return false;
+}
